packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -103,7 +103,7 @@ export function PanelChrome({
 
   return (
     <div className={styles.container} style={containerStyles} aria-label={ariaLabel}>
-      <div className={styles.loadingBarContainer}>
+      <div className={styles.loadingBarContainer} data-testid="panel-loading-wrapper">
```

Comment:
Do we call things wrappers by default? Why don't we call it just `panel-loading-indicator` and only represent semantically the object, not reflecting its internal layout of it?

Reply:
You are right, we don't need to semantically represent explicitly what the arrangement is, just if the required object is there. In general I prefer not to use `data-testid`, but `div` and `span` components are "invisible" to RTL selectors. 
On another note, this loading bar is not checked correctly anyway - the condition is after the wrapper. I'm changing to have custom aria-label and look for the label in the tests :)